### ISSUE TITLE
The option "Start when I log on" doesn't work

### ISSUE THREAD

**Issue by aaaaaa2493** [winsiderss/systeminformer#1851]
### Brief description of your issue

When I restart my laptop, the SystemInformer is not starting.
1. No icon in the system tray. 
2. When I manually start SystemInformer In the "System information" the graphs are just started logging info meaning they haven't logged memory and CPU beforehand, so no instance of System Informer were active previously.
3. In the Settings -> Apps -> Autorun the System Informer seems to be enabled
![изображение](https://github.com/winsiderss/systeminformer/assets/20189630/4031de62-8d37-47dc-906e-4a61cee47f13)
4. The option "Enable start as admin" makes no difference, the SystemInormer doesn't start regardless i this option is turned on or off.
![изображение](https://github.com/winsiderss/systeminformer/assets/20189630/03ab3f8f-7b30-48eb-9779-3b59dc0e6363)

### Environment (optional)

```shell
System Informer 3.0.7148 (ed40620)
Windows 11 Pro 22H2 22621.2283
```

**Comment by dmex** [comment]
Have you tried unchecking and rechecking the tickboxes?

**Comment by aaaaaa2493** [comment]
> Have you tried unchecking and rechecking the tickboxes?

@dmex Yes,I did. I've tried unchecking and rechecking "Start when I log in" as well as "Enable start as admin"
I tried 2 variants
1. First On second On
2. First On second Off

In both cases the SystemInformer is not starting

**Comment by CaCTuCaTu4ECKuu** [comment]
Had the same issue, can confirm.
I initially get portable version that, later updated it, but autorun didn't work until I manually reinstall app.
I make clean install (uninstall app from windows settings and then install it again) because it was eager to install where portable app was placed

**Comment by ge0rdi** [comment]
Do you have `System Informer` item with proper path to SI in `HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Run`?

Also note that `Run` key is processed by Windows some time after logon (like 30s or so).
So you will need to wait a bit after logon for SI to start.

**Comment by aaaaaa2493** [comment]
After updating to 24H2 everything started working. I was amazed to see system informer in tray where I usually don't see it after restaring my laptop, and it was right after installing update. Maybe it was some weird Windows bug or something
